tailcfg/tailcfg.go

```diff
@@ -147,7 +147,8 @@ type CapabilityVersion int
 //   - 102: 2024-07-12: NodeAttrDisableMagicSockCryptoRouting support
 //   - 103: 2024-07-24: Client supports NodeAttrDisableCaptivePortalDetection
 //   - 104: 2024-08-03: SelfNodeV6MasqAddrForThisPeer now works
-const CurrentCapabilityVersion CapabilityVersion = 104
+//   - 105: 2024-08-05: Fixed SSH behavior on systems that use busybox (issue #12849)
```

Comment:
I assume this is bumped so we can turn off the nodeattr to force the v1 behavior?

Reply:
Yes, although I don't intend to do so immediately. Preferably, some people who need V2 behavior use [this](https://github.com/tailscale/tailscale/pull/13048) for a while. If and when that looks good, we can remove the node attribute that forces V1.